Godel.Microservice.CurrencyRates/Models/CurrencyModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Godel.Microservice.CurrencyRates.Model
+{
+    public class CurrencyModel
+    {
+        public int Id { get; set; }
+
+        public string CurrencyName { get; set; }
```

Comment:
please remove Currency as it is already in class name

Reply:
was removed